pages/vi/firststeps.md

```diff
@@ -37,6 +37,7 @@ The goal of OLE is to provide educational infrastructure to learners and educato
 	- [Raspberry Pi Camera]("https://www.raspberrypi.org/products/camera-module-v2/")
 	- [Raspberry Pi Night Vision Camera]("https://www.raspberrypi.org/products/pi-noir-camera-v2/")
 	- [generic Raspberry Pi 5 Megapixel Camera]("https://www.amazon.com/AuviPal-Megapixels-Raspberry-Camera-Module/dp/B07PQ63D2S/")
+	- [Raspberry Serial Cable]("https://www.amazon.com/s?k=raspberry+pi+serial+cable")
```

Comment:
Somewhere along the line these links has quotes placed around them.
.md doesn't redirect to the link when they're in quotes. Remove them and the problem should be resolved.

Reply:
Looks like the quotes are still there.